DetectorDescription/DDCMS/test/python/dump.py

```diff
@@ -7,6 +7,11 @@
     input = cms.untracked.int32(1)
     )
 
+process.DDDetectorESProducer = cms.ESSource("DDDetectorESProducer",
+                                            confGeomXMLFiles = cms.string('DetectorDescription/DDCMS/data/cms-tracker.xml')
```

Comment:
@ianna this path is creating problems when one tries to execute the configuration out of the box

Reply:
it works for me:
```
$ cmsRun DetectorDescription/DDCMS/test/python/dump.py
Begin processing the 1st record. Run 1, Event 1, LumiSection 1 on stream 0 at 17-Dec-2018 18:51:44.344 CET
PersistencyIO    INFO  +++ Set Streamer to dd4hep::OpaqueDataBlock
CompactLoader    INFO  +++ Processing compact file: DetectorDescription/DDCMS/data/cms-tracker.xml with flag (null)
DD4CMS           INFO  +++ Processing the CMS detector description file:///build/yana/DD4HEP/CMSSW_10_4_X_2018-12-17-1100/src/DetectorDescription/DDCMS/data/cms-tracker.xml
Detector         WARN  +++ Object 'cms-tracker:zero' is already defined and new one will be ignored
DD4CMS           WARN  +++ Converter<ElementaryMaterial> No element present with name:materials:TD_Aluminium  [FAKE IT]
DD4CMS           WARN  +++ Converter<ElementaryMaterial> No element present with name:materials:Bor 10  [FAKE IT]
DD4CMS           WARN  +++ Converter<ElementaryMaterial> No element present with name:materials:Bor 11  [FAKE IT]
DD4CMS           WARN  +++ Converter<ElementaryMaterial> No element present with name:materials:Deuterium  [FAKE IT]
DD4CMS           WARN  +++ Converter<ElementaryMaterial> No element present with name:materials:Lutecium  [FAKE IT]
DD4CMS           WARN  +++ Converter<ElementaryMaterial> No element present with name:materials:Phosphor  [FAKE IT]
DD4CMS           WARN  +++ Composite material "materials:Water" not present!
DD4CMS           WARN  +++ Composite material "trackermaterial:T_Copper" not present!
DD4CMS           WARN  +++ Composite material "trackermaterial:T_Copper" not present!
DD4CMS           WARN  +++ Composite material "tobmaterial:TOB_DOH" not present!
Detector         INFO  *********** Created World volume with size:  500  500  500
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
******** My DDTECPhiAlgo!!!!!!
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
%MSG-w TIBGeom:  DDCMSDetector:test 17-Dec-2018 18:51:44 CET Run: 1 Event: 1
DOOHM placement sucks for Geant4. ERASED!
%MSG
DD4CMS           INFO  +++ Skip disabled algorithms: track:DDTOBRadCableAlgo
Detector         INFO  +++ Patching names of anonymous shapes....
DDDefinition     INFO  +++ Finished processing file:///build/yana/DD4HEP/CMSSW_10_4_X_2018-12-17-1100/src/DetectorDescription/DDCMS/data/cms-tracker.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/CMSCommonData/data/materials.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/CMSCommonData/data/rotations.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbarmaterial.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbarladder.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbarladderfull.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbarladderhalf.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbarlayer.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbarlayer0.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbarlayer1.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbarlayer2.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixbar.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/trackerpixbar.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/tracker.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/trackermaterial.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/TrackerCommonData/data/pixfwdMaterials.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/CMSCommonData/data/cmsMother.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/CMSCommonData/data/normal/cmsextent.xml
/cvmfs/cms-ib.cern.ch/nweek-02555/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_4_X_2018-12-17-1100/src/Geometry/CMSCommonData/data/cms.xml
DD Vector Registry size: 0
Iterate over the detectors:
world: /world
..done!
DDTestVectors::analyze:
DD Vector Registry size: 0
DDTestDumpFile::analyze:
In the DDTestDumpFile::analyze method...obtained main geometry, level=10

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 TIBGeom              -w DDCMSDetector:te                      16       16

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 TIBGeom              1/1              1/1              1/1

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                16                  16

dropped waiting message count 0
```